includes/html/graphs/device/ucd_memory.inc.php

```diff
@@ -14,6 +14,7 @@
 $rrd_options .= " 'DEF:ashared=$rrd_filename:shared:AVERAGE'";
 $rrd_options .= " 'DEF:abuffered=$rrd_filename:buffered:AVERAGE'";
 $rrd_options .= " 'DEF:acached=$rrd_filename:cached:AVERAGE'";
+$rrd_options .= " 'DEF:sysavail=$rrd_filename:sysavail:AVERAGE'";
```

Comment:
@murrant please take a look at this file change, I have zero experience in rrd options :)

Reply:
getting this ERROR: No DS called 'sysavail' in 'pve.tl.home/ucd_mem.rrd' from the `show RRD command`

Reply:
because your ucd_mem.rrd doesn't contain a sysavail data set ;)  You haven't added it yet.  And to merge this, we'll need it to not error in this case.

Reply:
right, I was expecting `data_update($device, 'ucd_mem', $tags, $fields);` would update the `.rrd` which didn't.

could you please guide how to make the sysavail available in that `.rrd` ?